src/datastore/common.ts

```diff
@@ -69,6 +69,9 @@ export interface DbTx {
   smart_contract_contract_id?: string;
   smart_contract_source_code?: string;
 
+  /** Only valid for `smart_contract` & `contract_call` tx types. */
```

Comment:
I don't think we really want this is a separate field in the db (it will never be used in a query)